src/gpu/intel/ocl/ocl_post_ops.h

```diff
@@ -188,7 +188,7 @@ float fwd_Xnary(bool is_binary, unsigned algorithm, float x, float y,
 #define APPLY_ALL_PO_STAGES(accumulator, acc_elem_dt, ...) \
     { \
         const int nelems = sizeof(accumulator) / sizeof(acc_elem_dt); \
-        acc_elem_dt *acc = &accumulator; \
+        acc_elem_dt *acc = (acc_elem_dt *)&accumulator; \
```

Comment:
Lets avoid performing a pointer cast here as it can suppress incorrect pointer conversions. Instead, lets fix the caller of this macro similar to the following:

```
APPLY_POST_OPS_SERIAL(&C[0][0], DATA_T, ...
```